### PR TITLE
Wikimedia:Linkshere implemented

### PR DESCRIPTION
**Pull request by djr-jsr**
Please Review @abinashmeher999

### CODE REVIEW

File: wiki_linkshere.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def get_linkshere(*pageids):
+    linkshere = {}
+    payload['pageids'] = '|'.join(str(x) for x in list(pageids))
+    res = requests.get(base_url, params=payload).json()
+    for pageid, page_content in res['query']['pages'].items():
+        if 'linkshere' not in page_content:
+            continue
+        linkshere[pageid] = linkshere.get(pageid, []) + [x['pageid'] for x in page_content['linkshere']]
```
Comment (abinashmeher999) [inline wiki_linkshere.py]: what is `x`? I would suggest to have a name that makes sense.

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]

**Comment by djr-jsr** [comment]
Anything else?

**Comment by abinashmeher999** [comment]
Currently it only returns a list of `pageids` right? That works.

**Comment by djr-jsr** [comment]
Yup. We can change that when we integrate it together.

**Comment by abinashmeher999** [comment]
Can you mention that in a `TODO` near that? This is so that we don't forget to include the `title`s too later when we integrate.

**Review by abinashmeher999 — APPROVED** [review]

**Comment by djr-jsr** [comment]
Here you go! 😄

**Pull request merged**